src/worker/ingestion/hooks.ts

```diff
@@ -0,0 +1,225 @@
+import { PluginEvent } from '@posthog/plugin-scaffold'
+import { captureException } from '@sentry/node'
+import { StatsD } from 'hot-shots'
+import fetch from 'node-fetch'
+import { format } from 'util'
+
+import { Action, Person } from '../../types'
+import { DB } from '../../utils/db/db'
+import { stringify } from '../../utils/utils'
+import { OrganizationManager } from './organization-manager'
+import { TeamManager } from './team-manager'
+
+export enum WebhookType {
+    Slack = 'slack',
+    Discord = 'discord',
```

Comment:
Is discord a new type? Don't see it here: https://github.com/PostHog/posthog/blob/master/posthog/tasks/webhooks.py#L110-L113

Reply:
Kind of, we don't need to explicitly handle it in control flows, but just to get things more in order (since we do mention it on the Project Settings page) I added detection of it. Also because that's what I use for testing all this webhook functionality :)